misc/testerina/modules/testerina-core/src/main/ballerina/Module.md

```diff
@@ -214,43 +226,39 @@ A custom mock object can be defined in place of the real object which should con
 
 _main.bal_
 ```ballerina
-http:Client clientEndpoint = new("http://petstore.com");
+http:Client clientEndpoint = check new ("http://petstore.com");
 
 function getPet(string petId) returns Pet | error {
-  http:Response|error result = clientEndpoint->get("/pets?id="+petId);
-  if(result is error) {
-    return result;
-  } else {
-      Pet pet = constructPetObj(result); 
-    return pet;
-  }
+  http:Response|error result = check clientEndpoint->get("/pets?id="+petId);
+  Pet pet = constructPetObj(result); 
```

Comment:
This section of `Object Mocking` has been updated to the current standard, however, the sample itself is incomplete. One cant simply copy this example and run as it would require some more object definitions and other functions to be defined

Do we proceed with the existing sample? Since we have examples elsewhere, shall we keep it as is?

Reply:
Shall we add a simple `constructPetObj` function and also any other missing functions/definitions? Let's use a complete sample. If you feel it's getting large, let's add another simple example. WDYT?

Reply:
I changed the example a bit to exclude this `constructPetObj`. I have no clue what the original function intended to do but the current example simply demonstrates how it works. 

We need to review these examples specifically.